lessons/Data-Science/AI/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,335 @@
+@Q{How are the diagrams we drew similar and how are they different?}
+}
+
+Let's imagine that our first round had started with the question, "Is it a utensil?" and had led us to "knife".  After the first round, our tree might have looked the diagram on the left (below). If the second round started with the same question, we could have just added to the original diagram... and we might have ended up with something like what you see on the right.
+
+[cols="^.2a,<.^1a,3a", grid="none", frame="none", stripes="none"]
+|===
+
+| @hspace{8em}**Round 1**
+
+|
+
+| @hspace{8em}**Round 2**
+
+| @image{images/tree1.png, 200}
+
+| @image{images/arrow.png, 50}
+
+| @image{images/tree2.png, 370}
+
+|===
+
+
+Notice that the topmost question—"is it a utensil?"—now splits left ("yes, it is a utensil") *and* right ("no, it is not a utensil").
+
+Our diagram begins with two unique pathways from the top of the tree to two unique items.
+
+Put another way: __There are two branches that split from the tree's root node. Each branch leads to a decision node, which eventually leads to a leaf node.__
+
+@lesson-instruction{
+Let's identify the root node, branches, decision nodes, and leaf nodes on our tree so far.
+}
+
+@teacher{Discuss your in-progress tree to help students locate the different parts. The parts are intuitively named, but formal definitions are below if needed.}
+
+@right{@image{images/terminology-tree.png, 300}}
+
+- The root node is the very top node that represents the entire population or sample.
+- @vocab{Splitting} is the process of dividing a node into sub-nodes with branches.
+- Decision nodes split from the root node, or from other nodes.
+- A leaf node is a node that does not split.
+
+@vspace{1ex}
+
+@lesson-instruction{
+- With a partner, turn to @printable-exercise{decision-tree.adoc} and complete the decision tree so that all six items are categorized.
+- Then, draw an entirely different decision tree - one which has a __different__ question at its root.
+}
+
+@teacher{
+
+@right{@image{images/tree4.png, 400}}
+
+One possible completed tree is on the right.
+
+To create a different tree, students can start with any of the questions from levels 2 or 3, or they can generate an entirely different starting question.
+
+As students finish, invite them to draw their trees on the board. Ideally, you will have a wide assortment of trees! If students are not developing interesting trees, urge them to think of *entirely* different questions than those posed in the sample tree.
+
+}
+
+@QandA{
+@Q{After looking at the decision trees of your classmates: What do these trees all have in common? How are they different?}
+@A{Answers will vary. Each tree will have twice as many branches as nodes. Many trees will have the same number of nodes, although probably not all. Many questions asked will likely be the same, but not every question.}
+}
+
+Let's take a step back and see how well some of our decision trees will perform.
```

Comment:
It might make sense to split the lesson here. This is a very long section and I would argue that from here forward you have new objectives. 

The first section could be called **intro to decision trees** and the second could be called something like **training a decision tree**